products/tests/test_views.py

```diff
@@ -0,0 +1,49 @@
+import json
+from rest_framework import status
+from django.test import TestCase, Client
+from django.urls import reverse
+from ..serializers import ProductSerializer
+from ..models import Product, ModelKey, Brand
+from IPython import embed
+
+from rest_framework.test import APIClient
+
+# initialize the APIClient app
+client = Client()
+api_client = APIClient()
+
+class GetAllProductsTest(TestCase):
+    """ Test module for GET all products API """
+    databases = {'products'}
```

Comment:
Is this used by the `TestCase`? Or is this informational?

Reply:
By `this` do you mean `databases = { ... }` part? In test setup, I am creating a few objects in products database to check if the endpoint is returning them.